app/controllers/matches_controller.rb

```diff
@@ -1,40 +1,40 @@
 # encoding: utf-8
 
 class MatchesController < ApplicationController
-  before_filter :require_league
+  before_action :require_league
   has_mobile_fu false
 
   def index
-    @date = params['date'] && Date.parse(params['date']) || Date.today
+    @date = match_params['date'] && Date.parse(match_params['date']) || Date.today
```

Comment:
Das mit den match_params ist eigentlich nur muss mass-assignments wie in create relevant, den Rest musste man nicht anpacken